expr/expr.go

```diff
@@ -143,18 +143,27 @@ type Masq struct {
 	RegProtoMax uint32
 }
 
+const (
+	// MasqRandom defines flag for a random masquerade
+	MasqRandom = 0x4
+	// MasqFullyRandom defines flag for a fully random masquerade
+	MasqFullyRandom = 0x10
+	// MasqPersistent defines flag for a persistent masquerade
+	MasqPersistent = 0x8
+)
```

Comment:
Ah, I didn’t know these weren’t available in x/sys/unix yet.

Can you make the names and definition match the `NF_NAT_RANGE_*` constants in `linux/netfilter/nf_nat.h` please?

Reply:
done, but names in nf_nat.f looks confusing as no visible relation to masq.

Reply:
Yeah, masq re-uses these flags under the hood apparently. Can you add a comment to switch to constants in the unix package once they become available?